build.gradle

```diff
@@ -40,6 +40,12 @@ compileTestJava {
   options.incremental = true
 }
 
+task testUnit(type: Test) {  
+  useJUnit {
+    includeCategories 'marquez.UnitTests'
+  }
+}
+
```

Comment:
This should probably go into a different review just b/c it's purpose is different than the rest of the changes.

Reply:
We use the `UnitTests` category below in `ConnectionUrlTest`, so it's (loosely) relevant to the change, but should have been introduced in a separate PR